content/03-reference/01-tools-and-interfaces/02-prisma-client/078-error-reference.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: 'Prisma Client error reference'
+metaTitle: 'Prisma Client error reference (Reference)'
+metaDescription: 'Prisma Client errors'
+---
+
+<TopBlock>This page describes Prisma Client exceptions and error codes.</TopBlock>
+
+## <inlinecode>PrismaClientKnownRequestError</inlinecode>
+
+Prisma Client throws a `PrismaClientKnownRequestError` exception if the query engine returns a known error related to the request - for example, a unique constraint violation.
+
+| **Property** | **Description**                                                                                                  |
+| :----------- | :--------------------------------------------------------------------------------------------------------------- |
+| `code`       | A Prisma-specific [error code](#error-codes).                                                                    |
+| `meta`       | Additional information about the error - for example, the field that caused the error: `{ target: [ 'email' ] }` |
+
+### Error codes
+
+#### Query Engine
+
+| **Error** | **Example message**                                                                                                                                           | **Description** |
+| :-------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------ | :-------------- |
+| `P2000`   | "The provided value for the column is too long for the column's type. Column: \${column_name}"<br />                                                          |
```

Comment:
```suggestion
| `P2000`   | The provided value for the column is too long for the column's type. Column: `${column_name}`<br />                                                          |
```

Reply:
These table are auto-generated by a script so I will need to edit that :)